src/domain/signupGroup/validation.ts

```diff
@@ -290,23 +291,22 @@ export const showErrors = ({
   }
 };
 
-const getFocusableFieldId = (
+type FieldList = Array<string | RegExp>;
+
+const isInFieldList = (fieldList: FieldList, fieldName: string): boolean =>
```

Comment:
This together with adding the new constant `SIGNUP_GROUP_FORM_CHECKBOX_GROUP_FIELDS` to `src/domain/signupGroup/constants.ts` could be in their own commit.